src/pages/guides/uxp_guide/uxp-misc/manifest-v5/index.md

```diff
@@ -0,0 +1,282 @@
+}
+```
+
+With manifest v5, the feature flag has been removed by v5 and the `launchProcess` permission must be declared to use `openExternal()` or `openPath()`. 
+```json
+"permissions": {        
+    "launchProcess": {    
+        // allows launching files with specified URI schemes         
+        "schemes": 
+            [ "https", "slack", "adbxd" ],   
+        // allows opening files with the specified file extensions          
+        "extensions": 
+            [ ".pdf", ".xd", ".ps" ],        
+    },   
+}
+```
+Both openPath() and openExternal() rely on this permission set, and upon either function call, the user will get a runtime consent dialog. Only after they agree will the API call execute. 
+
+### Plugin communication 
+If enabled, the plugin can communicate with other installed plugins. Defaults to `false`.
+```json
+"permissions": {        
+    "ipc": {
+        "enablePluginCommunication": true   
+    }
+}
+```
+```js
+const { pluginManager } = require("uxp");
+// find the Alchemist plugin in the loaded plugins
+const alchemistPlugin = Array.from(pluginManager.plugins).find(plugin => plugin.id==="2bcdb900");   
+//What commands and panels are available?
+const alchemistCommands = Array.from(alchemistPlugin.manifest.commands, command => command.commandId); // result: ["resetStateFn"]
+const alchemistPanels = Array.from(alchemistPlugin.manifest.panels, command => command.panelId); // result: ["inspector"]     
+// Show the inspector panel; note that panels can only be made visible -- you can't ask to hide the panel
+alchemistPlugin.showPanel("inspector");
+// Reset Alchemists state... but be sure you want to do this!
+alchemistPlugin.invokeCommand("resetStateFn");
+```
+
+### Host Communication 
+If enabled, the plugin can communicate with installed filters, automation plugins, etc. Defaults to `false`.
+```json
+"permissions": {        
+    "ipc": {
+        "enableHostCommunication": true   
+    }
+}
+```
+
+### Fonts 
+This permission allows the plugin to access the fonts installed on the system. 
+```json
+"permissions": {        
+    "fonts": "readInstalled" 
+}
+```
+
+## WebViews
+WebViews are available with UXP 6.0, and need to be configured in your plugin's manifest (v5 required).
+
+### Limitations 
+* Only available within modal dialogs for now.
+* Only remote content is allowed unless `webview.allowLocalRendering` is set to yes.
+
+In your `manifest.json`: 
+```json
+{
+    "manifestVersion": 5,
+    "featureFlags": {
+        "enableExperimentalWebviewSupport" : true
+    },
+    "requiredPermissions": {
+        "webview": {
+            "allow": "yes",
+            // domains --> string[] | "all"
+            "domains": [ "https://*.adobe.com", "https://*.google.com"],
+            "allowLocalRendering": "yes"
+        }
+    }
+}
+```
+Then in your plugin's code: 
+```html
+<dialog>
+    <webview id="webview" width="100%" height="360px" src="https://www.google.com"></webview>
```

Comment:
The screen shot below shows adobe.com. Can you change the src url in the example code?